src/app/core/data/data.service.ts

```diff
@@ -90,4 +101,15 @@ export abstract class DataService<TNormalized extends CacheableObject, TDomain>
     // return this.rdbService.buildSingle(href));
   }
 
+  findEmbedded<TEmbeddedNormalized extends CacheableObject, TEmbeddedDomain>(
```

Comment:
There is a `/collections` endpoint, so you should be able to do:
`http://dspace7.4science.it/dspace-spring-rest/#/dspace-spring-rest/api/core/collections?scope=7669c72a-3f2a-451f-a3b9-9210e7a4c02f` if the scope param isn't implemented on that endpoint on the rest side, we shouldn't work around it in the UI, but wait for it to be implemented.

I don't believe UI components should know about the way a resource is sent by the rest API, whether it's embedded or not, whether it needs to be fetched from a different URL. It should only know about the data services. If it needs a collection, it should address the collectionDataService and ask for it, everything else should be abstracted away, otherwise the code becomes too coupled.